server/store/store.go

```diff
@@ -921,6 +928,35 @@ func (s *SQLStore) CompareAndSetJobStatus(jobName string, oldStatus, newStatus b
 	return isUpdated, nil
 }
 
+func (s *SQLStore) GetStats() (*Stats, error) {
+	query := s.getQueryBuilder().Select("count(mmChannelID, mmTeamID) as linkedChannels").From("msteamssync_links").Limit(maxLimitForLinks)
```

Comment:
The `.Limit(maxLimitForLinks)` seems redundant here, given there will always be one row?

Reply:
Also, is`(mmChannelID, mmTeamID)` a compound primary key?

Reply:
Fixed all this